pgactivity/types.py

```diff
@@ -907,6 +907,7 @@ class BaseProcess:
     duration: Optional[float]
     state: str
     query: Optional[str]
+    encoding: str = attr.ib(converter=converters.default_if_none("utf-8"))  # type: ignore[misc]
```

Comment:
The default value should be `server_encoding`, which is usually `utf-8`.
Not sure it's worth changing.

Reply:
Indeed. I'll try to use that data later on (it's not easy now as we don't have the connection available at this point).